### ISSUE TITLE
JSON Fields aren't working anymore

### ISSUE THREAD

**Issue by bernhardh** [eminiarts/nova-tabs#140]
Since your last updates, JSON Fields (for example https://github.com/armincms/json field, but probably similar as well) are broken. I am getting `Undefined property: Armincms\Json\Json::$panel` in `vendor/eminiarts/nova-tabs/src/Tabs.php:50`:

```php
if ($field instanceof MergeValue) {
    if (null === $field->panel) {
        $field->panel = $this->name;
    }


    $this->addFields(
        new Tab($tab->getTitle(), $field->data)
    );

    continue;
}
```

MergeValues doesn't have to have defined `panel`'s, right?

**Comment by RVxLab** [comment]
Damn, didn't think of that one. This was a fix intended for #112.

A simple `property_exists` check should fix that. I'll fix it tonight and provide a 1.4.1 for it.

**Comment by bernhardh** [comment]
Correct. I have fixed it with `if ($field instanceof MergeValue && isset($field->panel)) {`. Should I do a PR for it or is it easier if you fix it yourself?

**Comment by RVxLab** [comment]
That line would undo the fix mentioned. I'll take care of it.

Thanks for the bug report and sorry about this.

**Comment by RVxLab** [comment]
I just released v1.4.1 to address this issue. Thanks again for your report.
